index.html

```diff
@@ -164,26 +195,29 @@ <h1 class="nameSalute">Hey There. I’m Praise</h1>
           />
         </div>
       </section>
-      <section class="workSection">
+      <section class="workSection" id="portfolio">
         <h1 class="workTitle">My Recent Works</h1>
+        <span class="border-through"></span>
         <div class="recentWork">
           <div class="recentWorkImg">
             <img src="./images/ImgPlaceholder.png" alt="" />
           </div>
-          <h2 class="font-32">Multi-Post Stories</h2>
-          <p class="font-16">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. has been the industry's standard dummy text ever
-            since the 1500s, when an unknown printer took a standard dummy text.
-          </p>
-          <div class="stackList">
-            <div class="stackBox">css</div>
-            <div class="stackBox">html</div>
-            <div class="stackBox">bootstrap</div>
-            <div class="stackBox">Ruby</div>
+          <div class="multi-post-txt-wrap">
+            <h2 class="font-32 mt-0">Multi-Post Stories</h2>
+            <p class="font-16">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <div class="stackList">
+              <div class="stackBox">css</div>
+              <div class="stackBox">html</div>
+              <div class="stackBox">bootstrap</div>
+              <div class="stackBox">Ruby</div>
+            </div>
+            <button class="btn-orange" type="button">See Project</button>
           </div>
```

Comment:
Please use the font color for the project title, paragraph, and technologies from the design page.